@ecomplus/widget-trustvox/cms.config.js

```diff
@@ -39,6 +39,11 @@ export default () => ({
           label: 'ID da loja na Trustvox',
           name: 'trustvoxStoreId',
           widget: 'number'
+        },
+        {
+          label: 'Link do script do widget',
+          name: 'widgetSrc',
+          widget: 'string'
```

Comment:
Campos no Netlify CMS são obrigatórios por padrão, em um campo como este é importante explicitar que é opcional, neste caso em específico também temos um valor padrão:

```suggestion
          widget: 'string',
          required: false,
          default: 'https://static.trustvox.com.br/assets/widget.js'
```